integration_tests/e2e/prisons/sessionTemplates/addSessionTemplate.cy.ts

```diff
@@ -1,214 +1,101 @@
-    cy.task('stubGetPrison', activePrison)
     cy.task('stubPrisons')
-    cy.task('stubGetSessionTemplates', { prisonCode, rangeType, sessionTemplates: [sessionTemplate] })
-    cy.task('stubIncentiveGroups', { prisonCode, body: [incentiveLevelGroupOne, incentiveLevelGroupTwo] })
-    cy.task('stubCategoryGroups', { prisonCode, body: [categoryGroupOne, categoryGroupTwo] })
-    cy.task('stubLocationGroups', { prisonCode, body: [locationGroupOne, locationGroupTwo] })
-  })
-
-  it('when on session template view clicking add session template should go to correct page', () => {
-    // Given
-    const viewSessionTemplatesPage = Page.createPage(ViewSessionTemplatesPage)
-    viewSessionTemplatesPage.goTo(prisonCode)
-
-    // When
-    viewSessionTemplatesPage.getAddSessionTemplateButton().click()
-
-    // Then
-    Page.verifyOnPage(AddSessionTemplatePage)
-  })
-
-  it('when on add session template page incentive groups are available', () => {
-    // Given
-    const addSessionTemplatePage = Page.createPage(AddSessionTemplatePage)
-
-    // When
-    addSessionTemplatePage.goTo(prisonCode)
-
-    // Then
-    addSessionTemplatePage
-      .getIncentiveGroupCheckBoxes()
-      .should('have.value', incentiveLevelGroupOne.reference, incentiveLevelGroupTwo.reference)
-  })
-
-  it('when on add session template page can select incentive groups', () => {
-    // Given
-    const addSessionTemplatePage = Page.createPage(AddSessionTemplatePage)
-
-    // When
-    addSessionTemplatePage.goTo(prisonCode)
-
-    // Then
-    addSessionTemplatePage.getHasIncentiveGroupsCheckBox().check()
-    addSessionTemplatePage
-      .getIncentiveGroupCheckBoxes()
-      .check([incentiveLevelGroupOne.reference, incentiveLevelGroupTwo.reference])
-  })
-
-  it('when on add session template page category groups are available', () => {
-    // Given
-    const addSessionTemplatePage = Page.createPage(AddSessionTemplatePage)
-
-    // When
-    addSessionTemplatePage.goTo(prisonCode)
-
-    // Then
-    addSessionTemplatePage
-      .getCategoryGroupCheckBoxes()
-      .should('have.value', categoryGroupOne.reference, categoryGroupTwo.reference)
-  })
-
-  it('when on add session template page can select category groups', () => {
-    // Given
-    const addSessionTemplatePage = Page.createPage(AddSessionTemplatePage)
-
-    // When
-    addSessionTemplatePage.goTo(prisonCode)
+    cy.signIn()
 
-    // Then
-    addSessionTemplatePage.getHasCategoryGroupsCheckBox().check()
-    addSessionTemplatePage.getCategoryGroupCheckBoxes().check([categoryGroupOne.reference, categoryGroupTwo.reference])
+    cy.task('stubGetPrison', TestData.prisonDto())
   })
 
-  it('when on add session template page location groups are available', () => {
-    // Given
-    const addSessionTemplatePage = Page.createPage(AddSessionTemplatePage)
-
-    // When
-    addSessionTemplatePage.goTo(prisonCode)
+  it('should add a session template from the listings page', () => {
```

Comment:
Consolidated lots of individual tests for specific form fields to one test that checks them in one go.